src/command_modules/azure-cli-cognitiveservices/azure/cli/command_modules/cognitiveservices/commands.py

```diff
@@ -29,4 +29,5 @@ def load_command_table(self, _):
 
     # deprecating this
     with self.command_group('cognitiveservices') as g:
-        g.custom_command('list', 'list_resources', deprecate_info='az cognitiveservices account list')
+        g.custom_command('list', 'list_resources',
+                         deprecate_info=g.deprecate(redirect='az cognitiveservices account list', expiration='2.1.0'))
```

Comment:
My recommendation would be to hide this. You can keep the expiration if you want to, but hiding the command will make it not discoverable via help, which will help prevent newcomers form relying on it. Existing scripts that use it of course will still work.

Reply:
Ya it should be better to hide, changed to hide.